bluebutton-data-pipeline-fhir-load/src/test/resources/logback-test.xml

```diff
@@ -12,7 +12,7 @@
 	<!-- At 'debug', this will ensure that request and response headers are logged. 
 		At 'trace', this will ensure that request and response bodies are logged. -->
 	<logger name="gov.hhs.cms.bluebutton.datapipeline.fhir.load.FhirLoader"
-		level="info" />
+		level="trace" />
```

Comment:
Do we want this to be there always, or just as needed? I've been turning it down when I needed it, and reverting it before I committed, but maybe that doesn't make sense?